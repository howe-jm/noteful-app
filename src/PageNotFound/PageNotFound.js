import React from 'react';
import ErrorPage from '../ErrorBoundary/ErrorBoundary';
import ApiContext from '../ApiContext';
import './PageNotFound.css';

export default class PageNotFound extends React.Component {
  static contextType = ApiContext;

  render() {
    return (
      <ErrorPage>
        <div className='not-found'>
          <h2>Note not found!</h2>
          <p>Get outta here, stalker.</p>
        </div>
      </ErrorPage>
    );
  }
}
