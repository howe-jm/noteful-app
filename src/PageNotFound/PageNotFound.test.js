import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PageNotFound from './PageNotFound';

describe(`Add Folder component`, () => {
  it('Renders without errors', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
