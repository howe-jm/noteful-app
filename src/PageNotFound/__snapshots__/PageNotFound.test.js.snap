// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Folder component Renders without errors 1`] = `
<ErrorPage>
  <div
    className="not-found"
  >
    <h2>
      Note not found!
    </h2>
    <p>
      Get outta here, stalker.
    </p>
  </div>
</ErrorPage>
`;
